client/src/views/AboutView.vue

```diff
@@ -1,15 +1,145 @@
 <template>
-  <div class="about">
-    <h1>This is an about page</h1>
-  </div>
+  <v-img :src="CSFBanner" width="100%"></v-img>
+  <v-row style="background-color: #202020" class="pt-8 px-8" no-gutters justify="center">
+    <h2 class="text-secondaryTint text-center">
+      <span class="font-weight-bold">Education changes lives:</span>
+      we are challenging all of you to let your kilometres make a difference!
+    </h2>
+  </v-row>
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320">
+    <path
+      fill="#202020"
+      d="m0 64 60 10.7c60 10.3 180 32.3 300 48 120 16.3 240 26.3 360 16C840 128 960 96 1080 85.3c120-10.3 240-.3 300 5.4l60 5.3V0H0Z"
+    />
+  </svg>
+  <v-img :src="StrideForEducationRBGLogo" class="px-8 mt-n8 mb-8" width="100%"></v-img>
+  <v-divider class="mx-4" />
+  <p class="pa-8">
+    Community Spirit Foundation is running Stride For Education – a challenge designed to motivate
+    kids across Australia to get active while learning about how our foundation supports First
+    Nations young people.
+  </p>
+  <p class="px-8 pb-8">
+    Clock as many kilometres as you can with your team or as an individual! Stay informed and
+    motivated by checking the leaderboard, where you can see how your performance stacks up against
+    other participants. Compare your progress with other users, teams, or even your own teammates,
+    and let the friendly competition inspire you to reach new levels of achievement.
+  </p>
+  <v-img :src="StrideForEducationGif" width="100%"></v-img>
+  <p class="pa-8">
+    Every step you take as part of this challenge is one step further in Community Spirit
+    Foundation's journey of helping to support First Nations young people to realise their potential
+    through lifelong learning.
+  </p>
+  <!-- TODO: Use CSF for button -->
+  <v-row justify="end" class="px-8 pb-4" no-gutters>
+    <v-btn
+      class="bg-primaryRed pb-14"
+      size="x-large"
+      variant="flat"
+      :style="{ fontFamily: 'Hackney', fontSize: '42px' }"
```

Comment:
I know there will be a better way of styling every button with the same font but I can't find a solid answer for it.

Reply:
You could use https://vuetifyjs.com/en/styles/text-and-typography/ typography helper classes to control size of text but if this is applied then the font-family doesn't apply and it defaults to the Material Design font, Roboto.